src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockPresenter.java

```diff
@@ -0,0 +1,14 @@
+package use_cases.display_availability_timeblocks;
+
+public class DisplayAvailabilityTimeBlockPresenter implements DisplayAvailabilityTimeBlockOutputBoundary {
+
+    @Override
+    public String availabilityTimeBlockDisplayed(DisplayAvailabilityTimeBlockModel displayedAvailabilityTimeBlock) {
+        return displayedAvailabilityTimeBlock.toString();
```

Comment:
I notice that the toString() method in the model isn't overridden, so this most definitely won't return what you are looking for.